hw/formal/tools/jaspergold/fpv.tcl

```diff
@@ -83,6 +83,14 @@ if {$env(DUT_TOP) == "rv_dm"} {
   clock clk_rd_i -factor 3
   clock -rate {rvalid_o, rready_i, rdata_o} clk_rd_i
   reset -expr {!rst_ni}
+
+} elseif {$env(DUT_TOP) == "pinmux_tb"} {
+  clock clk_i -both_edges
+  clock clk_aon_i
+  clock -rate {alert_tx_i, tl_i, usb_suspend_i, usb_aon_wake_ack_i, usb_aon_wake_en_i,
+               usb_out_of_rst_i, sleep_en_i} clk_aon_i
```

Comment:
I believe all these signals are actually running on the normal clock (`clk_i`) and are synchronized to `clk_aon_i` internally, if needed.

The only interface signals that I am aware of that are running on `clk_aon_i` are output signals, for instance:
https://github.com/lowRISC/opentitan/blob/8b675981e9a9f2a7ebadc2f7964e4b2437bacce6/hw/ip/pinmux/rtl/pinmux.sv#L27-L29

Reply:
Thanks for pointing that out! Let me try delete line 90 and 91 and see if that affects any assertions!

Reply:
It works! Just somehow the run time is longer :) But thanks again for pointing that out!